README.md

```diff
@@ -304,43 +252,32 @@ Kway Chap is a Teochew dish of flat, broad rice sheets in a soup made with dark
 
 [Pictures of Kway Chap via Google](https://www.google.com.sg/search?q=kway+chap+singapore&es_sm=91&source=lnms&tbm=isch&sa=X&ei=rRiqU7e_CsmHuAS3yYGQBA&ved=0CAgQ_AUoAQ&biw=1916&bih=986)
 
-#### Where can I find it?
-
-##### Lao San Kway Chap
-
-Address: 232 Ang Mo Kio Ave 3, #01-1222, Singapore 560232 [Map](https://goo.gl/maps/vq4gNeDrUeE2)
-
 ### Chilli Crab
 
 Singapore is very famous for its chilli crab.
 
-If you can get there
-[Mellben](https://www.facebook.com/MellbenSeafoodToaPayoh) is quite good. Try
-their Crab Bee Hoon.]
+[Jumbo Seafood](https://www.jumboseafood.com.sg/en/home) is classic in a nice setting, but can be expensive. Ask for prices before you order.
 
-[Comment: Don't try this at touristy restaurants - it's just not worth it.]
+[Mellben](https://www.mellbensignature.sg/mellben-seafood/) is a more local option, albeit a bit out of the way. Try their Crab Bee Hoon if you go there. 
 
 ### Wanton Noodles
 
-Again, not a truly local dish, but a very nice Chinese dish of noodles and
-pork.
-
 Worth visiting [Foong Kee Coffee Shop](http://www.hungrygowhere.com/singapore/foong_kee/)
-if you're in Chinatown.
+in Chinatown.
 
 ![Wanton Min](/images/wanton_min.jpg)
 
 ### Putu Piring / Kueh Tutu
 
-Putu Piring is a Traditional Singaporean treat. Steamed rice cake filled with gula melaka (melted palm sugar). Served with grated coconut, squares of pandan leaf, has subtle, sweet vanilla fragrance.
+Putu Piring is steamed rice cake filled with gula melaka (melted palm sugar). Served with grated coconut, squares of pandan leaf, has subtle, sweet vanilla fragrance.
 
-## Indie Bookstore
+## Indie Bookstores
 
 - [BooksActually](https://www.booksactuallyshop.com/pages/about)
```

Comment:
Booksactually is closed. this link goes to a gambling site. I think we should remove this section.